proc/api.thrift

```diff
@@ -0,0 +1,40 @@
+service Proc {
+  
+  exception InvalidRequest {
+    1: optional string message
+  }
+
+  exception ScootTooBusy {
```

Comment:
Maybe "CanNotSchedule"? E.g., maybe Scoot isn't that busy, but it wants to rate limit just you?